controllers/apply-next/form-router.js

```diff
@@ -313,24 +144,8 @@ function initFormRouter(formModel) {
     });
 
     formModel.sections.forEach((sectionModel, sectionIndex) => {
-        /**
-         * Route: Form sections
-         */
         router.get(`/${sectionModel.slug}`, (req, res) => {
-            const sectionLocalised = res.locals.form.sections.find(s => s.slug === sectionModel.slug);
-            if (sectionLocalised.summary) {
-                res.locals.breadcrumbs = concat(res.locals.breadcrumbs, {
-                    label: sectionLocalised.title
-                });
-                res.render(path.resolve(__dirname, './views/section-summary'), {
-                    title: `${sectionLocalised.title} | ${res.locals.form.title}`,
-                    section: sectionLocalised,
-                    backUrl: null, // @TODO: Determine backUrl
-                    nextUrl: `${req.baseUrl}/${sectionModel.slug}/1`
-                });
-            } else {
-                res.redirect(`${req.baseUrl}/${sectionModel.slug}/1`);
-            }
+            res.redirect(`${req.baseUrl}/${sectionModel.slug}/1`);
```

Comment:
Does this mean we're dropping the concept of a summary "step" for a given section?

Reply:
I think so for now, going to come back to it when we revisit the beneficiaries screen. It felt like it could go on the first screen in the step.